### PR TITLE
Advanced cryptogaphy comment

### LINKED ISSUE #2398: Remove references to crypto_valid_advanced?
Since support for cryptography < 2.0 tests were removed in #2260, should we remove references to `crypto_valid_advanced` at these locations?
```
$ git grep crypto_valid_advanced
scapy/config.py:    crypto_valid_advanced = isCryptographyAdvanced()
scapy/layers/tls/automaton_cli.py:            if conf.crypto_valid_advanced:
scapy/layers/tls/automaton_cli.py:                            if conf.crypto_valid_advanced:
scapy/layers/tls/automaton_cli.py:        if conf.crypto_valid_advanced:
scapy/layers/tls/crypto/cipher_aead.py:if conf.crypto_valid_advanced:
scapy/layers/tls/crypto/cipher_aead.py:        # if conf.crypto_valid_advanced:
scapy/layers/tls/crypto/cipher_aead.py:if conf.crypto_valid_advanced:
scapy/layers/tls/crypto/cipher_aead.py:            if (conf.crypto_valid_advanced and
scapy/layers/tls/crypto/cipher_aead.py:                if (conf.crypto_valid_advanced and
scapy/layers/tls/crypto/cipher_aead.py:                    if (conf.crypto_valid_advanced and
scapy/layers/tls/crypto/cipher_aead.py:if conf.crypto_valid_advanced:
scapy/layers/tls/crypto/cipher_aead.py:        # if conf.crypto_valid_advanced:
scapy/layers/tls/crypto/cipher_aead.py:if conf.crypto_valid_advanced:
scapy/layers/tls/keyexchange_tls13.py:if conf.crypto_valid_advanced:
scapy/layers/tls/keyexchange_tls13.py:                if conf.crypto_valid_advanced:
scapy/layers/tls/keyexchange_tls13.py:                if conf.crypto_valid_advanced:
scapy/layers/tls/record.py:if conf.crypto_valid_advanced:
scapy/layers/tls/record.py:            if (conf.crypto_valid_advanced and
```

### PR DESCRIPTION
**Pull request by guedou**
This PR fixes #2398

### CODE REVIEW

File: scapy/config.py [hunk 1]
```diff
@@ -378,8 +378,8 @@ def _version_checker(module, minver):
 
 def isCryptographyValid():
     """
-    Check if the cryptography library is present, and if it is recent enough
-    for most usages in scapy (v1.7 or later).
+    Check if the cryptography module >= 1.7 is present. This is the minimum
```
Comment (gpotter2) [inline scapy/config.py:381]: Actually we've dropped <2.0.0
https://github.com/secdev/scapy/commit/b6589098cca0fda17efa9e1b0181365b53128196
The PR was using some >2.0.0 only things.

Comment (guedou) [reply scapy/config.py:381]: What should I do? Remove this function?

Comment (gpotter2) [reply scapy/config.py:381]: The function still checks if cryptography is present. We could issue a warning if the version is < 2.0.0 (and update it to return True only when > 2.0.0)

Comment (guedou) [reply scapy/config.py]: I updated the PR.

**Pull request merged**